### DISCUSSION TITLE
Can a non-root user use --bind to map /data/debug/sandbox to /sandbox

### DISCUSSION THREAD

**Discussion by zhangvv123** [netblue30/firejail#6573] [Q&A]
Can a non-root user use bind to map `/data/debug/sandbox` to `/sandbox`? My user is `worker`. Can I implement firejail execution by adding authorization to the `worker` user?

**Comment by rusty-snake** [comment]
I think we can implement arbitrary bind mounts for unprivileged users as long as we set `PR_NO_NEW_PRIVS`.
